src/main/java/ru/yandex/qatools/embed/postgresql/PostgresProcess.java

```diff
@@ -145,9 +149,11 @@ protected void onBeforeProcess(IRuntimeConfig runtimeConfig)
             throws IOException {
         List<String> ret = new ArrayList<>();
         ret.addAll(asList(exe.executable().getAbsolutePath(),
-                "-h", config.net().host(),
-                "-p", String.valueOf(config.net().port()),
-                "-D", config.storage().dbDir().getAbsolutePath()
+                "start",
+                "-o",
+                String.format("\"-p %s\"", String.valueOf(config.net().port())),
```

Comment:
fixed host param so it works both on win and linux (before it worked just on win)